server/bundler/bin/bundler.js

```diff
@@ -106,4 +106,15 @@ webpack( webpackConfig, function( error, stats ) {
 	assets = utils.getAssets( stats.toJson() );
 
 	fs.writeFileSync( path.join( __dirname, '..', 'assets.json' ), JSON.stringify( assets, null, '\t' ) );
+
+	if ( ! process.env.WEBPACK_OUTPUT_JSON ) {
```

Comment:
ah, I was looking at the subset. Yeah, this just speeds up minification when we're using the external procs to do it. Looks good.